README.md

```diff
@@ -36,13 +36,13 @@ Table of Contents
 
 The deployment supports a range of Kubernetes cluster versions:
 
-* 1.25 - 1.29
+* 1.28 - 1.30
 
 The range comes via the dependency images and their supporter versions:
 
-* Cert-Manager: 1.24 - 1.29
+* Cert-Manager: 1.24 - 1.30
 * Node Feature Discovery: 1.21 ->
-* Intel Device Plugins: 1.25 - 1.29
+* Intel Device Plugins: 1.28 - 1.30
```

Comment:
Btw. It may be better to add also note to "Disabling dependencies" section that if dependencies are removed from _values.yaml_, it may be advisable to remove relevant ones also from _cleanup.sh_, to avoid potential mishaps later on...